lldb/test/API/commands/plugin/TestPlugin.py

```diff
@@ -82,10 +82,10 @@ def test_completions(self):
         )
 
         # A completion for a full namespace should contain the plugins in that namespace.
-        self.completions_contain("plugin list abi", ["abi.sysv-x86_64"])
-        self.completions_contain("plugin list abi.", ["abi.sysv-x86_64"])
-        self.completions_contain("plugin list abi.s", ["abi.sysv-x86_64"])
-        self.completions_contain("plugin list abi.sysv-x", ["abi.sysv-x86_64"])
+        self.completions_contain("plugin list object-file", ["object-file.JSON"])
```

Comment:
The object file Json plugin will always be there? 

Can we gate the test in such a way to ensure that?

Reply:
I do not see anything in the CMakeLists.txt file that conditionally enables the JSON object file plugin (unlike the ABI plugins). I also do not see any gates in the json object file tests:

https://github.com/llvm/llvm-project/blob/bc187b82705f31b5cd04fe986170f594ab2a51d4/lldb/test/API/functionalities/json/object-file/TestObjectFileJSON.py#L13

So I think it should always be there.